docs: create documentation for jira service management contact point (#100444)
grafana/grafana: docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-jira.md

---
canonical: https://grafana.com/docs/grafana/latest/alerting/configure-notifications/manage-contact-points/integrations/configure-jira/

> description: Configure the Jira Service Management integration to receive notifications when your alerts are firing

Reviewer: ```suggestion
description: Configure the Jira integration to receive notifications when your alerts are firing
```

keywords:
  - grafana
  - alerting
  - Jira

>   - Jira Service Management

Reviewer: ```suggestion
```

  - integration
labels:
  products:
    - cloud
    - enterprise
    - oss

> menuTitle: Jira Service Management

Reviewer: ```suggestion
menuTitle: Jira
```

Reviewer: I tested the integration with their Cloud version. 

Jira is the brand name of the main product (Jira platform). 

Jira Service Management is a Jira App that creates Jira projects tailored for "service management and support". 

The alerting integration uses the REST API and creates "issues" in Jira projects using the REST API, supporting Jira Service Management and other types of Jira projects.

Because of this, I suggest naming the integration as `Jira` instead of `Jira Service Management`.

> title: Configure Jira Service Management for Alerting

Reviewer: ```suggestion
title: Configure Jira for Alerting
```

weight: 121
refs:
  configure-contact-points:
    - pattern: /docs/grafana/
      destination: /docs/grafana/<GRAFANA_VERSION>/alerting/configure-notifications/manage-contact-points/
    - pattern: /docs/grafana-cloud/
      destination: /docs/grafana-cloud/alerting-and-irm/alerting/configure-notifications/manage-contact-points/
---

> # Configure Jira Service Management for Alerting

Reviewer: ```suggestion
# Configure Jira for Alerting
```

Use the Jira integration in a contact point to create issues in your Jira instance when alerts fire. The integration supports both Jira Cloud and Jira Server/Data Center installations.

## Before you begin

Before you begin, ensure you have the following:

- A Jira instance (Cloud, Server, or Data Center)
- API access credentials for Jira
- Appropriate permissions to create issues in your target Jira project

## Configure Jira for a contact point

To create a contact point with a Jira integration, complete the following steps:

1. Navigate to **Alerts & IRM** -> **Alerting** -> **Contact points**.
2. Click **+ Add contact point**.
3. Enter a name for the contact point.
4. From the **Integration** list, select **Jira**.
5. Set up the required [settings](#required-settings) for your Jira configuration.
6. Click **Save contact point**.

For more details on contact points, including how to test them and enable notifications, refer to [Configure contact points](/docs/grafana/latest/alerting/configure-notifications/manage-contact-points/).

### Required Settings

| Key | Description | 
| --- | --- |

> | URL | The URL of your Jira instance (e.g., `https://your-domain.atlassian.net` or `https://jira.your-domain.com`). |

Reviewer: ```suggestion
| URL | The URL of the REST API of your Jira instance. Supported versions: `2` and `3` (e.g., `https://your-domain.atlassian.net/rest/api/3`). |
```

> | Basic Auth User | Username for authentication. |

Reviewer: ```suggestion
| Basic Auth User | Username for authentication. For Jira Cloud, use your email address. |
```

| Basic Auth Password | Password or personal token. For Jira Cloud, you need to obtain a personal token [here](https://id.atlassian.com/manage-profile/security/api-tokens)) and use it as the password. |
| API Token | An alternative to basic authentication, a bearer token is used to authorize the API requests. See [Jira documentation](https://confluence.atlassian.com/enterprise/using-personal-access-tokens-1026032365.html) for more information. |

> | Project Key | The project key where issues are be created (e.g., `ALERTS`). |

Reviewer: ```suggestion
| Project Key | The project key identifying the project where issues will be created. Project keys are unique identifiers for a project. |
```

> | Basic Auth Password | Password or personal token. For Jira Cloud, you need to obtain a personal token [here](https://id.atlassian.com/manage-profile/security/api-tokens)) and use it as the password.                                                       |

Reviewer: ```suggestion
| Basic Auth Password | Password or personal token. For Jira Cloud, you need to obtain a personal token [here](https://id.atlassian.com/manage-profile/security/api-tokens) and use it as the password.                                                       |
```
